<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ai-Client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>winicon.ico</normaloff>winicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="scrollArea_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>501</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>499</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>510</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="userEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>701</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Heebo</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKey_Setting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionModify_Key"/>
   </widget>
   <addaction name="menuKey_Setting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionModify_Key">
   <property name="text">
    <string>Modify Key</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
